### PR TITLE
Updated logic to cater for manually entered cqcregistered workplace

### PR DESCRIPTION
**Pull request by tgreyuk**

### CODE REVIEW

**Comment by tgreyuk** [comment]
[(3) Registration - CQC Registered Account that has changed their Address](https://trello.com/c/Uc1q6md3/30-3-registration-cqc-registered-account-that-has-changed-their-address)

File: src/app/features/registration/select-main-service/select-main-service.component.ts [hunk 1]
```diff
@@ -29,12 +28,8 @@ export class SelectMainServiceComponent extends SelectMainService {
...
   }
 
-  protected getSelectedLocation(): void {
-    this.subscriptions.add(
-      this.registrationService.selectedLocationAddress$.subscribe((location: LocationAddress) =>
-        this.getServicesByCategory(location)
-      )
-    );
+  protected getServiceCategories(): void {
+    this.getServicesByCategory(this.registrationService.isRegulated());
```
Comment (omarCreativeDev) [inline src/app/features/registration/select-main-service/select-main-service.component.ts]: what about handling the unsubscription?

File: src/app/features/add-workplace/select-main-service/select-main-service.component.ts [hunk 2]
```diff
@@ -27,12 +26,8 @@ export class SelectMainServiceComponent extends SelectMainService {
...
   }
 
-  protected getSelectedLocation(): void {
-    this.subscriptions.add(
-      this.workplaceService.selectedLocationAddress$.subscribe((location: LocationAddress) =>
-        this.getServicesByCategory(location)
-      )
-    );
+  protected getServiceCategories(): void {
+    this.getServicesByCategory(this.workplaceService.isRegulated());
```
Comment (omarCreativeDev) [inline src/app/features/add-workplace/select-main-service/select-main-service.component.ts]: what about handling the unsubscription?

File: src/app/features/registration/select-main-service/select-main-service.component.ts (hunk 1 above)
Comment (tgreyuk) [reply src/app/features/registration/select-main-service/select-main-service.component.ts]: Unsubscribed

File: src/app/features/add-workplace/select-main-service/select-main-service.component.ts (hunk 2 above)
Comment (tgreyuk) [reply src/app/features/add-workplace/select-main-service/select-main-service.component.ts]: Unsubscribed

**Pull request merged**